src/main/java/com/bankingservice/banking/exception/ErrorCode.java

```diff
@@ -0,0 +1,31 @@
+package com.bankingservice.banking.exception;
+
+import lombok.Getter;
+import lombok.ToString;
+
+@Getter
+@ToString
+public enum ErrorCode {
+    /**
+     * The user id not found
+     */
```

Comment:
1. Comments for ErrorCodes is not required.
2. Include the HttpResponseStatus in these codes